sdk/core/azure-core/tests/test_rest_http_request.py

```diff
@@ -309,4 +310,11 @@ def test_use_custom_json_encoder():
 #             b"<file content 2>\r\n",
 #             b"--+++--\r\n",
 #         ]
-#     )
\ No newline at end of file
+#     )
+
+
+def test_backcompat_mixin():
```

Comment:
This is the test I added to make sure the new request has all of the attrs of the old